src/layouts/EditPage.jsx

```diff
@@ -367,55 +372,65 @@ export default class EditPage extends Component {
-                    action: async () => {
-                      this.setState({ isSelectingImage: true });
-                    },
-                    className: 'fa fa-picture-o',
-                    title: 'Insert Image',
-                    default: true,
-                  },
-                  {
-                    name: 'link',
-                    action: async () => { 
-                      this.onHyperlinkOpen() 
-                    },
-                    className: 'fa fa-link',
-                    title: 'Insert Link',
-                    default: true,
-                  },
-                  tableButton,
-                  guideButton,
-                ],
-              }}
-            />
-          </div>
+          {
+            resourceType !== 'file'
+            ?
+              <div className={`${editorStyles.pageEditorSidebar} ${isLoadingPageContent ? editorStyles.pageEditorSidebarLoading : null}`} >
+                {
+                  isLoadingPageContent
+                  ? (
+                    <div className={`spinner-border text-primary ${editorStyles.sidebarLoadingIcon}`} />
+                  ) : ''
+                }
+                <SimpleMDE
+                  id="simplemde-editor"
+                  className="h-100"
+                  onChange={this.onEditorChange}
+                  ref={this.mdeRef}
+                  value={editorValue}
+                  options={{
+                    toolbar: [
+                      headingButton,
+                      boldButton,
+                      italicButton,
+                      strikethroughButton,
+                      '|',
+                      codeButton,
+                      quoteButton,
+                      unorderedListButton,
+                      orderedListButton,
+                      '|',
+                      {
+                        name: 'image',
+                        action: async () => {
+                          this.setState({ isSelectingImage: true });
+                        },
+                        className: 'fa fa-picture-o',
+                        title: 'Insert Image',
+                        default: true,
+                      },
+                      {
+                        name: 'link',
+                        action: async () => { 
+                          this.onHyperlinkOpen() 
+                        },
+                        className: 'fa fa-link',
+                        title: 'Insert Link',
+                        default: true,
+                      },
+                      tableButton,
+                      guideButton,
+                    ],
+                  }}
+                />
+              </div>
+            :
+              <div className={`${editorStyles.pageEditorSidebar}`}>
+                <div className={`text-center`}>
+                  Editing is disabled for downloadable files.
+                </div>
+              </div>
```

Comment:
What do you think about something similar to the loading styles but with lower opacity? My concern is that I still want to preserve the format of the `EditPage` and that removing the text editor is a different view that introduces a small bit of cognitive load. Let me know what you think

<img width="1064" alt="Screenshot 2020-12-16 at 3 21 26 PM" src="https://user-images.githubusercontent.com/31984694/102317427-784d6a00-3fb2-11eb-85fb-bbf5ba52ab15.png">

Reply:
Makes sense, made the change in https://github.com/isomerpages/isomercms-frontend/pull/274/commits/4964587a7b727403ea0aff55908f40658890fb38